box-office.rb

```diff
@@ -1,5 +1,12 @@
 require 'sinatra'
+require 'httparty'
 
 get '/' do
   'Hello world!'
 end
+
+get '/movies/:imdbid' do |id|
+  response = HTTParty.get("http://www.omdbapi.com/?apikey=#{ENV['OMDB_API_KEY']}&i=#{id}&plot=full&r=json")
```

Comment:
This is probably not good, if we're using dotenv this will most likely change. And also @server-monitor brought up that we should extract all client calls as a separate interface and require it in box-office.